Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -461,24 +444,24 @@ def report_results(self, virtualdrive) -> int:
 class VirtualDriveManager(object):

```

Comment:
Worth updating name to `WindowsVirtualDriveManager` to distinguish it from the Linux one (and not that it is a base implementation)?

Reply:
Updated